diesel/src/pg/types/floats/mod.rs

```diff
@@ -41,12 +41,12 @@ impl Error for InvalidNumericSign {
 impl FromSql<types::Numeric, Pg> for PgNumeric {
     fn from_sql(bytes: Option<&[u8]>) -> Result<Self, Box<Error+Send+Sync>> {
         let mut bytes = not_none!(bytes);
-        let ndigits = try!(bytes.read_u16::<NetworkEndian>());
```

Comment:
I named this variable for parity with the pg code which is the "documentation" for this structure. I don't feel strongly on whether we need to keep it or not.